update.sh

```diff
@@ -51,13 +51,15 @@ fi
 
 pushd alpine
    url=$(get_url Dockerfile amd64)
+   echo $url
```

Comment:
Is this (and the others below it) a debug leftover?

Reply:
I think this is useful to leave in - re-running to get that piece of info can be avoided.